bdns/dns.go

```diff
@@ -150,11 +151,13 @@ var (
 	}
 )
 
+type ResolverAddr string
```

Comment:
Since LookupHost can have multiple results, we should consider renaming is to `ResolverAddrs []string` and use the stringer to concatenate the results.

```suggestion
type ResolverAddrs string
```

Reply:
Done